### PR TITLE
Adds the React take-home-interview

### PR DESCRIPTION
**Pull request by lucantini**

### CODE REVIEW

File: react_project/readme.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Carta - Frontend take-home interview assignment
+
+## Group Purchase store
```
[the remark this answers is not in the document]
Comment (lucantini) [reply react_project/readme.md]: Good catch! Gonna update it!

[the remark this answers is not in the document]
Comment (lucantini) [reply react_project/readme.md]: Good catch! Gonna update it!

File: react_project/readme.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- Quality Control - does it have it's quality assured with unit tests;
+
+#### Considerations
+The endpoints that will be used are:
+- User details: [https://api.github.com/users/{username}](https://developer.github.com/v3/users/)
+- Repos of a specific user: [https://api.github.com/users/{username}/repos](https://developer.github.com/v3/repos/#list-user-repositories)
+- Repo details: [https://api.github.com/repos/{full_name}](https://developer.github.com/v3/repos/#get)
+### Project instructions
+- Fork this repository on your machine
+- Remove the `.git` directory in the repo's root directory
```
Comment (rbusquet) [inline react_project/readme.md]: why do we need this step?

File: react_project/readme.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- Create a search input for GitHub user;
+- Show the searched user info - avatar, followers, following, e-mail and bio;
+- Show the searched user repos. Listed in order by number of stars;
+- Create the behavior to change the ordering of the listed repos - order by name/date;
+- Once a specific repo is clicked, the route changes, showing a page for the specific repo, with
+it's details - repo name, description, number of stars, language and a link to the GitHub repo page;
+
+#### Other Details
+- Using React is **mandatory**
+- Using routes is **mandatory**
```
Comment (rbusquet) [inline react_project/readme.md]: I think this could be a "senior" recomendation

File: react_project/readme.md (hunk 2 above)
Comment (lucantini) [reply react_project/readme.md]: Sorry! Got from the template we had, must have forgotten to remove!

**Pull request merged**